apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -560,6 +588,16 @@ contract DisputableVoting is DisputableAragonApp, IForwarder {
         emit ChangeOverruleWindow(_overruleWindow);
     }
 
+    /**
+    * @dev Internal function to change the execution delay
+    * @param _executionDelay New execution delay in seconds
+    */
+    function _changeExecutionDelay(uint64 _executionDelay) internal {
+        require(_executionDelay <= voteTime, ERROR_INVALID_EXECUTION_DELAY);
```

Comment:
Not sure if this condition should exist; theoretically it shouldn't matter how long your execution delay is for.